app/code/Magento/Directory/Setup/Patch/Data/AddDataForAustralia.php

```diff
@@ -29,28 +30,43 @@ class AddDataForAustralia implements DataPatchInterface, PatchVersionInterface
     private $dataInstallerFactory;
 
     /**
+     * @var \Magento\Directory\Model\ResourceModel\Region\CollectionFactory
+     */
+    private $regionCollectionFactory;
+
+    /**
+     * AddDataForAustralia constructor.
+     *
      * @param ModuleDataSetupInterface $moduleDataSetup
      * @param \Magento\Directory\Setup\DataInstallerFactory $dataInstallerFactory
+     * @param \Magento\Directory\Model\ResourceModel\Region\CollectionFactory $regionCollectionFactory
      */
     public function __construct(
         ModuleDataSetupInterface $moduleDataSetup,
-        \Magento\Directory\Setup\DataInstallerFactory $dataInstallerFactory
+        \Magento\Directory\Setup\DataInstallerFactory $dataInstallerFactory,
+        CollectionFactory $regionCollectionFactory
```

Comment:
Please, inject new dependencies according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)